src/platform/user/profile/vap-svc/actions/transactions.js

```diff
@@ -203,12 +203,14 @@ export function createTransaction(
         transaction,
       });
     } catch (error) {
+      const [firstError = {}] = error.errors ?? [];
```

Comment:
I admit that this is not the prettiest thing in the world. But it just ensures that `firstError` will be set to an empty object if the `error.errors` array is nullish.